AzureCalling/Views/ParticipantView.swift

```diff
@@ -42,6 +43,10 @@ class ParticipantView: UIView {
         participantLabel.isHidden = displayName.isEmpty
     }
 
+    func updateMuteIndicator(isMuted: Bool) {
+        participantMuteIndicator.isHidden = !isMuted
```

Comment:
Let's open an api in ParticipantLabelView to updateMuteIndicator. 
And call `participantLabelView.updateMutesIndicator`  here